client/grpc/block.go

```diff
@@ -0,0 +1,19 @@
+package tmservice
```

Comment:
maybe this could go inside `client/grpc/tmservice/block.go` (same forall other files)